code/src/components/StartScreen.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import games, { generateGame } from '../reducers/games'
+
+const StartScreen = () => {
+  const [inputValue, setInputValue] = useState('')
+
+  const dispatch = useDispatch()
+
+  const onUsernameSet = () => {
+    dispatch(games.actions.setUsername(inputValue))
+    dispatch(generateGame())
+} 
+
+  return (
+    <section className='start-game'>
```

Comment:
It is not nice that you forgot about the form tag. It striped your input from submission on enter, what is the most used sending the form method 😭